chapter16_tensor_methods/tensor_basics.ipynb

```diff
@@ -647,15 +599,13 @@
    "source": [
     "#### Tensor times vector\n",
     "\n",
-    "Similarly, for a vector of size 4 and contract along the mode 1."
+    "Similarly, we can contract along the mode 1 with a vector of size 4 (our tensor is of size (3, 4, 2).\n"
    ]
```

Comment:
Should be "we can contract along the first mode" or "we can contract along mode 1".

Reply:
Fixed. I hesitated to use first mode (which implies a zeroth mode) but settled with mode 1.